circleci/table_circleci_project.go

```diff
@@ -0,0 +1,102 @@
+package circleci
+
+import (
+	"context"
+	"regexp"
+	"strings"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableCircleciProject() *plugin.Table {
+	return &plugin.Table{
+		Name:        "circleci_project",
+		Description: "A CircleCI project shares the name of the code repository for which it automates workflows, tests, and deployment.",
+		List: &plugin.ListConfig{
+			Hydrate: listCircleciProjects,
+		},
+
+		Columns: []*plugin.Column{
+			{Name: "slug", Description: "A unique identification for the project in the form of: <vcs_type>/<org_name>/<repo_name> .", Type: proto.ColumnType_STRING},
+			{Name: "organization_slug", Description: "Organization that pipeline belongs to, in the form of: <vcs_type>/<org_name> .", Type: proto.ColumnType_STRING},
+			{Name: "username", Description: "Organization or person's username who owns the repository.", Type: proto.ColumnType_STRING},
+			{Name: "reponame", Description: "Name of the repository the project represent", Type: proto.ColumnType_STRING},
```

Comment:
```suggestion
			{Name: "reponame", Description: "Name of the repository the project represents", Type: proto.ColumnType_STRING},
```

Should this column be renamed to `repository_name`?